editor/plugins/sprite_frames_editor_plugin.cpp

```diff
@@ -821,19 +821,30 @@ void SpriteFramesEditor::_update_library(bool p_skip_selector) {
 
 	for (int i = 0; i < frames->get_frame_count(edited_anim); i++) {
 		String name;
-		Ref<Texture2D> icon;
+		Ref<Texture> frame = frames->get_frame(edited_anim, i);
```

Comment:
`SpriteFrames::get_frame()` does return `Texture2D`, this change was wrong.

Reply:
Oh, sorry, forgot about that change in `master`. New PR I guess?